### DISCUSSION TITLE
Albums preview links lack thumbnails

### DISCUSSION THREAD

**Discussion by dror3go** [photoprism/photoprism#3452] [General]
I'm not sure if this is a bug, a change in config that I've missed, a change in behavior or what, but in the past when I shared a link to an album in WhatsApp/Telegram/etc. There was a preview with thumbnails of several pictures from the album.
Somewhere in the past few releases this changed, and the preview shows only the album's name.

Is that the new desired behavior?

**Comment by lastzero** [comment]
I've also noticed that. Going to take a look when working in sharing again next time.

**Reply by kvalev** [reply]
The issue (at least for me) was the double slashes in the url, the fix is straightforward: https://github.com/kvalev/photoprism/commit/ca85a3c2b5427ea0ce06e70041d8fc3a7c7ab0a0

**Comment by dror3go** [comment]
@lastzero Should we keep this as a discussion, or would it be better to open a bug on the issue?

**Reply by graciousgrey** [reply]
I just created an issue to better keep track of this: https://github.com/photoprism/photoprism/issues/3658

**Reply by lastzero** [reply]
@dror3go It was just a `/` too much in the generated image URL. I've started a new development preview build so you can test the fix!
